packages/opentelemetry-core/src/trace/TracerDelegate.ts

```diff
@@ -29,20 +29,20 @@ export class TracerDelegate implements types.Tracer {
   // Wrap a tracer with a TracerDelegate. Provided tracer becomes the default
   // fallback tracer for when a global tracer has not been initialized
   constructor(
-    private readonly tracer: types.Tracer | null = null,
-    private readonly fallbackTracer: types.Tracer = new NoopTracer()
+    private readonly _tracer: types.Tracer | null = null,
```

Comment:
I remember a comment previously about it not be desirable to have arguments underscore prefixed. Not sure if that's enforceable in the linter. Doing so requires creating a temporary variable that is just in the argument rather than using something like `private readonly ...`.

cc/ @rochdev who had mentioned this previously

Reply:
I opened a separate issue: https://github.com/open-telemetry/opentelemetry-js/issues/274